### ISSUE TITLE
Handle nonce for submitting extrinsics to relay chain

### ISSUE THREAD

**Issue by shawntabrizi** [shawntabrizi/polkadot-launch#4]
Right now we dont really handle the nonce when submitting potentially multiple extrinsics in a block.

This is avoided by having a `sleep(6000)`, but it would be better to simply track the nonce internally.

Cant use `nextNonce` api, since that is not on all Substrate chains.

**Comment by seunlanlege** [comment]
currently blocked on this issue:

```
--- Submitting extrinsic to set balance of 5Ec4AhPUwPeyTFyuhGuBbD224mY85LKLMSqSSo33JYWCazU4 to 1000000000000000000000. (nonce: 1) ---
2022-01-31 11:36:45        RPC-CORE: submitAndWatchExtrinsic(extrinsic: Extrinsic): ExtrinsicStatus:: 1014: Priority is too low: (12482 vs 12482): The transaction has too low priority to replace another transaction already in the pool.
2022-01-31 11:36:45             DRR: 1014: Priority is too low: (12482 vs 12482): The transaction has too low priority to replace another transaction already in the pool.
2022-01-31 11:36:45        RPC-CORE: submitAndWatchExtrinsic(extrinsic: Extrinsic): ExtrinsicStatus:: 1014: Priority is too low: (12482 vs 12482): The transaction has too low priority to replace another transaction already in the pool.

```

**Comment by ltfschoen** [comment]
> currently blocked on this issue:
> 
> ```
> --- Submitting extrinsic to set balance of 5Ec4AhPUwPeyTFyuhGuBbD224mY85LKLMSqSSo33JYWCazU4 to 1000000000000000000000. (nonce: 1) ---
> 2022-01-31 11:36:45        RPC-CORE: submitAndWatchExtrinsic(extrinsic: Extrinsic): ExtrinsicStatus:: 1014: Priority is too low: (12482 vs 12482): The transaction has too low priority to replace another transaction already in the pool.
> 2022-01-31 11:36:45             DRR: 1014: Priority is too low: (12482 vs 12482): The transaction has too low priority to replace another transaction already in the pool.
> 2022-01-31 11:36:45        RPC-CORE: submitAndWatchExtrinsic(extrinsic: Extrinsic): ExtrinsicStatus:: 1014: Priority is too low: (12482 vs 12482): The transaction has too low priority to replace another transaction already in the pool.
> ```

is your relay chain finalising blocks?

**Comment by seunlanlege** [comment]
yes it is, had to remove the parachain balance so i don't run into this issue

**Comment by ltfschoen** [comment]
> yes it is, had to remove the parachain balance so i don't run into this issue

so you just removed this line from the parachain?
i tried doing that but i still get the same error when i run the "Make transfer and listen to events" script after connecting to my local parachain that's running locally https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9999#/js

**Comment by seunlanlege** [comment]
You might have to read the polkadot launch code, so you can see how to avoid this issue, that's what I did

**Comment by dzmitry-lahoda** [comment]
if zombienet handles that than sure need to switch
